src/main/java/com/ghilly/transformer/TransformerDAODTO.java

```diff
@@ -0,0 +1,32 @@
+package com.ghilly.transformer;
+
+import com.ghilly.model.DAO.CityDAO;
+import com.ghilly.model.DAO.CountryDAO;
+import com.ghilly.model.DTO.CityDTO;
+import com.ghilly.model.DTO.CountryDTO;
+
+import java.util.ArrayList;
+
+public class TransformerDAODTO {
+
+    private TransformerDAODTO() {
```

Comment:
add prefix and  postfix otherwise the name is so difficult for reading.
for example
fromDAOtoDTO